easybuild/easyconfigs/x/XZ/XZ-5.2.2-intel-2016a.eb

```diff
@@ -13,9 +13,6 @@ source_urls = ['http://tukaani.org/xz/']
 
 builddependencies = [
     ('Autotools', '20150215'),
-]
-
-dependencies = [
```

Comment:
If it's there, sure, but what if it's not there (older glibc)?

I wouldn't change this in this PR, and certainly not for just a single XZ easyconfig.